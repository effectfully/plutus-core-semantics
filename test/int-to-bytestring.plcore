(program

(module Foo (imported) (exported () ()) (local () ())
  (define bs1 (builtin intToByteString 1128))
  (define bs2 (builtin intToByteString 9))
  (define bs3 (builtin intToByteString 15))
  (define bs4 (builtin intToByteString 16))
  (define bs5 (builtin intToByteString 17))
  (define bs6 (builtin intToByteString 37858))
  (define bs7 (builtin intToByteString 0)))

)
