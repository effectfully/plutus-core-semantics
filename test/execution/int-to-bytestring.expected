<T>
  <k> . </k>
  <lenv> .Map </lenv>
  <genv> ( Foo . bs7 ) |-> #0 ( Foo . bs5 ) |-> #11 ( Foo . bs6 ) |-> #93E2 ( Foo . bs3 ) |-> #F ( Foo . bs4 ) |-> #10 ( Foo . bs1 ) |-> #468 ( Foo . bs2 ) |-> #9 </genv>
  <modules>
    <Module>
      <curr> 0 </curr>
      <name> Foo </name>
      <dataCons> .TyExpList </dataCons>
      <decl> .NameList </decl>
      <importedModules> .ModNmList </importedModules>
      <exportTy> .TyExpList </exportTy>
      <exportTm> .NameList </exportTm>
    </Module>
  </modules>
</T>
